sigs/storage-charter.md

```diff
@@ -0,0 +1,140 @@
+cloud-native environments through:
+
+
+
+1. Providing valuable and objective information to the TOC, End Users
+   and Projects of the CNCF regarding areas considered in scope (see
+   above).
+2. Collaborating effectively with other related groups (see below).
+3. Helping to maintain the continued health of CNCF Storage Projects
+   (see below)
+4. Identifying and filling gaps in the landscape of CNCF Storage
+   Projects.
+
+
+# Current CNCF Storage Projects
+
+
+
+1. TiKV
+2. etcd
+3. Vitess
+4. Rook
+5. [TBD] OpenEBS
+
+
+# Interfaces With Other Related Groups
+
+
+
+*   **Kubernetes Storage SIG** - is focussed towards
+    Kubernetes-specific storage abstractions, interfaces, and
+    implementations of these interfaces.  We maintain close
+    communication with this Kubernetes SIG, with several individuals
+    actively involved in both.  Our aim is to avoid unnecessary
+    duplication of effort by the two groups, and maintain clear an
+    consistent messaging by the two groups to our end user community
+    and projects.
+*   **CSI** - is focussed on defining an industry standard “Container
+    Storage Interface” (CSI) that will enable storage vendors to
+    develop a plugin once and have it work across a number of
+    container orchestration systems.  Again, we maintain close
+    communication with this group, and avoid unnecessary duplication
+    of effort and inconsistent messaging wherever possible.
+*   **CNCF Security SIG** - works on the more general area of
+    cloud-native security including authentication, authorization,
+    encryption, accounting, auditing and related topics.  We defer as
+    much as possible to this group to deal with general
+    security-related issues, and liaise closely with them on how to
+    deal with storage-specific security areas where these arise.
+*   **CNCF Apps SIG** (not yet fully formed) - will be focussed on the
+    development, deployment, operation and testing of cloud-native
+    applications.  We collaborate with this SIG where this pertains to
+    Storage.
+*   **K8s Apps SIG** - has done some work on how Kubernetes apps use
+    storage, as well as how storage systems (including databases) may
+    be deployed on Kubernetes .  We collaborate with Apps SIG and make
+    sure that important topics are well covered.
+*   **[Kubernetes Service Catalog SIG
+    ](https://github.com/kubernetes/community/tree/master/sig-service-catalog)**-
+    works on enabling external managed software offerings such as
+    datastore services offered by public cloud providers.
+
+
+# Operating Model
+
+This SIG follows the [standard operating
+guidelines](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#operating-model)
+provided by the TOC unless otherwise stated here.
+
+**Current TOC Liaison:** Xiang Li
+
+**[Proposed] Co-Chairs:** Alex Chircop, Quinton Hoole, “One Other TBD”
+
+**[Proposed] Tech Leads:** Saad Ali, Xing Yang, “One Other TBD”
```

Comment:
Add Sugu Sougoumarane as Tech Lead

Reply:
can you please remove [Proposed] -- when merged, that will ratify the co-chairs, right?  unless you plan on separate process for finalizing chairs :)

Reply:
yes - once the vote is complete, we will remove the [Proposed]